src/test/java/org/reaktivity/specification/mqtt/internal/MqttFunctionsTest.java

```diff
@@ -112,6 +112,28 @@ public void shouldEncodeMqttBeginExtAsSubscribe()
         assertEquals(1, mqttBeginEx.subscriptionId());
     }
 
+    @Test
+    public void shouldEncodeMqttBeginExtAsSubscribeWithOptions()
```

Comment:
Please rename `shouldEncodeMqttBeginExtAsSubscribeWithOptions()` to `shouldEncodeMqttBeginExtAsSubscribeWithFlags()` now that we've renamed the field and enums to `flags`.